server/id/id.go

```diff
@@ -119,13 +130,13 @@ func (alloc *allocatorImpl) rebaseLocked() error {
 		return errs.ErrEtcdTxnConflict.FastGenByArgs()
 	}
 
-	log.Info("idAllocator allocates a new id", zap.Uint64("alloc-id", end))
-	idallocGauge.Set(float64(end))
+	log.Info("idAllocator allocates a new id", zap.String("label", alloc.label), zap.Uint64("alloc-id", end))
+	alloc.metric.Set(float64(end))
```

Comment:
Could you enhance the readable? maybe like` alloc.metrics.idallocgauge.Set(float64(end))`, and also helpful for adding more metrics for each allocator.